### DISCUSSION TITLE
Bázis

### DISCUSSION THREAD

**Discussion by qbeer** [csabaiBio/ai4covid#1] [General]
Ez egy új feature itt a githubon, próbáljuk ki.

**Comment by qbeer** [comment]
Kaptok erről emailt?

**Reply by qbeer** [reply]
Nem, a watch-nál lehet be kell állítani a Custom menüpont alatt a Discussion-t.

**Reply by bedohazizsolt** [reply]
Működik.

**Reply by icsabai** [reply]
Nekem be van állítva mindenre a watch, meg is kaptam.

**Comment by qbeer** [comment]
Sziasztok,

 @bsulyok légyszíves töltsd fel az eddigi notebookokat.

Alex

**Comment by qbeer** [comment]
Sziasztok,

 ahogy beszéltük, kérlek ma délutánra töltsétek fel az eddigi adatspecifikus kódokat, én feldolgoztam a brixia modellt, elkezdtem az implementációt, de a pipeline kialakításához már kellenek a meglévő kódok. Közben egy volt évfolyamtársam megnézett egy egyszerű modellt, ami a szervezők cikkében is szerepelt, átlagok pótlásával az excel fileban és ~70% körüli cross-validációs pontosságot ért el. Ez lenne a baseline, majd a kódját migrálom megmutatni.

@bsulyok @bedohazizsolt @abiricz 

Alex

**Reply by qbeer** [reply]
+1, holnap délután lesz egy Q&A szekció a szervezőktől: 

COVID CXR Hackathon
Wednesday, February 16⋅14:00 – 15:00
[https://unimore-it.zoom.us/j/83618893771?pwd=WlEweTF0NkN1cGpLcDFFUllkNUlLZz09](https://www.google.com/url?q=https://unimore-it.zoom.us/j/83618893771?pwd%3DWlEweTF0NkN1cGpLcDFFUllkNUlLZz09&sa=D&source=calendar&usd=2&usg=AOvVaw3YjSIbny1b_sBI_W6IQFWr)

**Comment by icsabai** [comment]
Melyik szerveren megy ez? Az adatok le vannak töltve? A táblázatos adatokra esetleg ránézek én is, ha lesz egy kis időm.

**Reply by bedohazizsolt** [reply]
János szerver, /mnt/ncshare/ai4covid_hackathon/ alatt vannak az adatok.

**Comment by icsabai** [comment]
A mai meetingen amiket említettem:
- az "ősi" Netflix challenge iterative sparse PCA: https://sifter.org/~simon/journal/20061211.html
- kerestem python implementációt:  https://github.com/gbolmier/funk-svd
- amit mondtam még, hogy Börcsök Judit használt ilyet, az közben rájöttem, hogy más volt, nem missing value

**Comment by qbeer** [comment]
Valaki ír egy kiértékelő kódot?

https://github.com/csabaiBio/ai4covid/blob/0e2be09ad4b19f58592623b51603a849330c536f/run_inference.py#L52

Most már van egy predikciós script és jó lenne letesztelni, hogy biztos jól prediktál-e. A validációs split-ek megvannak hozzá a data/preprocessed_data/tables almappákban, szóval ki lehet próbálni, csak a csv-ket ne generáljátok újra. 👯 

Egyébként, valakinek szintén kellené még modularizálni a baseline scriptet, de erről beszéljünk holnap.

Alex
